modules/bytes/parser_factory.lua

```diff
@@ -129,31 +129,55 @@ function parser_factory.switch_resource(ready_payload, final_mess, index, tag_fi
     index = index + ready_payload.byte_offset
   end
   if ready_payload.name == 'uint' then
+    if index + math.ceil(ready_payload.size/8) > #message then
+      final_mess["error"] = 'Insufficient bytes available for your definitions'
+      return index
+    end
```

Comment:
since this code is used in many place, u can write a common function and reuse it

Reply:
@lenaw  thank you for suggestion, I wrote a common function on the last commit